lib/logstash/outputs/dynatrace.rb

```diff
@@ -61,7 +61,7 @@ def register
 
       def headers
         {
-          'User-Agent' => "logstash-output-dynatrace v#{PLUGIN_VERSION}",
+          'User-Agent' => "logstash-output-dynatrace/#{PLUGIN_VERSION}",
```

Comment:
You'll need to update the tests as well:
https://github.com/dynatrace-oss/logstash-output-dynatrace/blob/main/spec/outputs/dynatrace_spec.rb#L82